apps/speed.c

```diff
@@ -1033,11 +1033,12 @@ static int ECDSA_verify_loop(void *args)
     }
     return count;
 }
-
+/* TBC: outlen is globally shared, in a strange way*/
```

Comment:
Fixes are on my patch queue, I will PR it soon.
The next variable also should be pushed into `loopargs_t`